src/test/java/edu/ksu/canvas/attendance/controller/CourseConfigurationControllerITest.java

```diff
@@ -63,10 +63,10 @@ public void additionalSetup() throws NoLtiSessionException {
 
 
     @Test
-    public void classSetup_nonExistantSectionId() throws Exception {
-        Long nonExistantSectionId = 2000L;
+    public void classSetup_nonExistentSectionId() throws Exception {
+        Long nonExistentSectionId = 2000L;
         
-        mockMvc.perform(get("/courseConfiguration/"+nonExistantSectionId))
+        mockMvc.perform(get("/courseConfiguration/"+nonExistentSectionId))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/courseConfiguration/" 9 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)